pkg/tests/cross-tests/rapid_tv_gen.go

```diff
@@ -209,6 +209,7 @@ func (tvg *tvGen) GenBlockWithDepth(depth int) *rapid.Generator[tb] {
 				return tftypes.NewValue(objType, fields)
 			})
 		} else {
+			_ = rapid.Bool().Draw(t, "consumeBitstream1")
```

Comment:
This is really weird also. I think perhaps issue could be with this code when minFields=3 and there's no entropy going into the result? 

```
nFields := rapid.IntRange(minFields, 3).Draw(t, "nFields")
```

Reply:
I believe the issue comes when drawing the values, not the schema.

I suspect if we run into this branch, we produce a `rapid.Just(tftypes.NewValue(objType, map[string]tftypes.Value{}))` which consumes none of the entropy.

I don't see why this has to consume the bitstream, really - am I misunderstanding the framework here?

Reply:
The whole thing is a bit tricky since I haven't found a reliable way of reproducing the issue - it's sensitive to code changes, so the same seed does not produce the error after changing the code.

Reply:
rapid.Just should be fine so long as something else drew entropy. I think the problem arises where `rapid.Custom(f)` draws no entropy in `f`.

Reply:
We always draw entropy, when calling `rapid.IntRange(minFields, 3).Draw(t, "nFields")`. @VenelinMartinov If we leave a hack like this in, can you please mark it with a TODO and create an issue to track removing the hack.

That would be a good issue for `help-wanted` and `good-first-issue`.

Reply:
> That would be a good issue for help-wanted and good-first-issue.

Well FWIW I do not think so, unfortunately I do not think this would work very well.. As an external contributor I'd expect to have zero interest in verification work as it is too indirectly connected to end-user problems and outcomes. Perhaps we can make a distinction between easy onboarding tasks and welcoming external contributors.